components/compliance-service/ingest/server/compliance.go

```diff
@@ -35,78 +29,33 @@ type ComplianceIngestServer struct {
-	automateURL string, notifierClient notifier.Notifier, authzProjectsClient iam_v2.ProjectsClient,
-	eventServiceClient automate_event.EventServiceClient, configManager *config.ConfigManager) *ComplianceIngestServer {
+	automateURL string, notifierClient notifier.Notifier, authzProjectsClient iam_v2.ProjectsClient) *ComplianceIngestServer {
 
 	compliancePipeline := pipeline.NewCompliancePipeline(esClient, authzProjectsClient, mgrClient)
 
-	updateManager := project_update_lib.NewDomainProjectUpdateManager(esClient, authzProjectsClient, eventServiceClient,
-		configManager, event_ids.ComplianceInspecReportProducerID)
-
 	return &ComplianceIngestServer{
 		compliancePipeline: compliancePipeline,
 		client:             esClient,
 		mgrClient:          mgrClient,
 		automateURL:        automateURL,
 		notifierClient:     notifierClient,
-		updateManager:      updateManager,
 	}
 }
 
 func (srv *ComplianceIngestServer) HandleEvent(ctx context.Context, req *automate_event.EventMsg) (*automate_event.EventResponse, error) {
 	log.Debugf("compliance ingest is handling your event %s", req.EventID)
 
-	response := &automate_event.EventResponse{}
-	if req.Type.Name == event.ProjectRulesUpdate {
-		projectUpdateID, err := getProjectUpdateID(req)
-		if err != nil {
-			logrus.Errorf("Project Rule Update sent without a ProjectUpdateID eventID %q",
-				req.EventID)
-			return response, err
-		}
-
-		srv.updateManager.Start(projectUpdateID)
-	} else if req.Type.Name == event.ProjectRulesCancelUpdate {
-		projectUpdateID, err := getProjectUpdateID(req)
-		if err != nil {
-			logrus.Errorf("Project Rule Update Cancel sent without a ProjectUpdateID. eventID %q",
-				req.EventID)
-			return response, err
-		}
-
-		srv.updateManager.Cancel(projectUpdateID)
-	}
-
-	return response, nil
+	return nil, status.Error(codes.Unimplemented, "Unimplemented")
 }
 
 func (srv *ComplianceIngestServer) ProjectUpdateStatus(ctx context.Context,
 	req *ingest_api.ProjectUpdateStatusReq) (*ingest_api.ProjectUpdateStatusResp, error) {
-	time, err := ptypes.TimestampProto(srv.updateManager.EstimatedTimeComplete())
-	if err != nil {
-		log.Errorf("Could not convert EstimatedTimeComplete to protobuf Timestamp %v", err)
-		time = &tspb.Timestamp{}
-	}
-	return &ingest_api.ProjectUpdateStatusResp{
-		State:                 srv.updateManager.State(),
-		PercentageComplete:    srv.updateManager.PercentageComplete(),
-		EstimatedTimeComplete: time,
-	}, nil
-}
-
-func getProjectUpdateID(event *automate_event.EventMsg) (string, error) {
-	if event.Data != nil && event.Data.Fields != nil && event.Data.Fields["ProjectUpdateID"] != nil &&
-		event.Data.Fields["ProjectUpdateID"].GetStringValue() != "" {
-		return event.Data.Fields["ProjectUpdateID"].GetStringValue(), nil
-	}
-
-	return "", fmt.Errorf("Project Rule Update sent without a ProjectUpdateID eventID: %q", event.EventID)
+	return nil, status.Error(codes.Unimplemented, "Endpoint no longer used")
```

Comment:
Uhm... I can imagine situations where we'd care. However, don't we still deploy all services in one go, etc, and can thus just remove the method?

If we in fact **cannot**, we could also embed [`UnimplementedComplianceIngesterServer`](https://github.com/chef/automate/blob/d4e9c336fa73c8ebabf53043e13fdf4bcb91766e/components/compliance-service/ingest/ingest/compliance.pb.go#L218-L230) in `ComplianceIngestServer`, a nice little addition brought by the lastest go-grpc bump.

Reply:
neat